helpdesk_mgmt_fieldservice/views/res_partner.xml

```diff
@@ -1,17 +1,34 @@
 <odoo>
-
     <record id="res_partner_form_ticket_context" model="ir.ui.view">
         <field name="name">res.partner.form.ticket.context</field>
         <field name="model">res.partner</field>
-        <field name="groups_id" eval="[(4, ref('fieldservice.group_fsm_user_own'))]" />
         <field name="inherit_id" ref="base.view_partner_form" />
         <field name="arch" type="xml">
-            <button name="action_view_helpdesk_tickets" position="attributes">
+              <button name="action_view_helpdesk_tickets" position="before">
+                 <field name="service_location_id" invisible="1" />
+              </button>
+              <button name="action_view_helpdesk_tickets" position="attributes">
```

Comment:
**chore**: an issue that we should fix is that the `<button name="action_view_helpdesk_tickets">` is not present in `base.view_partner_form` but is added by `helpdesk_mgmt.view_partner_form` so we should inherit from that view

Reply:
Yes, this has to be inherited from helpdesk_mgmt.view_partner_form which is the right way to inherit as mentioned by you. 

However, I see issue still persists with updating button with group attribute to "fieldservice.group_fsm_user_own" and having helpdesk_mgmt and fieldservice modules installed along with helpdesk_mgmt_fieldservice with user provided only with helpdesk_mgmt but not with fieldservice, helpdesk button is not visible in the contacts view as shown below:

![image](https://github.com/user-attachments/assets/db35d10b-82af-48ae-afc5-e0c4250c009f)

![image](https://github.com/user-attachments/assets/15af3221-e2b6-48d1-88be-c08f2595626c)

Please let me know your views on it.